.gitignore

```diff
@@ -88,4 +88,5 @@ pyocd/_version.py
 pyocd.yaml
 pyocd_user.py
 
-
+# Mac stuff
+.DS_Store
```

Comment:
Normally `.DS_Store` is ignored in your `~/.gitignore_global` file. Is it not there on your system? (I'm not sure how that gets installed.)

Reply:
Sorry, I did not know that. I will revert that change.

Reply:
It might be set up that way only for certain installation methods of git, i.e. brew versus macports. Or it might have been set up on my system by the Fork UI.

Reply:
I had to do the following first and then edit the `~/.gitignore_global`  file and then reload VSCode. Thanks for letting me know. It did not work without doing the following
```sh
git config --global core.excludesfile ~/.gitignore_global 
```